core/controllers/editor_test.py

```diff
@@ -27,11 +27,12 @@
 from core.domain import event_services
 from core.domain import exp_domain
 from core.domain import exp_services
-from core.domain import stats_jobs_continuous_test
 from core.domain import rights_manager
+from core.domain import stats_jobs_continuous_test
 from core.domain import user_services
 from core.platform import models
-from core.platform.taskqueue import gae_taskqueue_services as taskqueue_services
+from core.platform.taskqueue import \
```

Comment:
Nope -- never end lines with \. See our coding guidelines. The previous way was fine, I think (and, if not, a different workaround is needed).

Reply:
Okay, got it! I will change the length of import lines according to our coding guidelines.